package/src/steps/IframeView/BodyIframeView.tsx

```diff
@@ -87,7 +92,41 @@ const BodyIframeView: React.FC<BodyIframeViewType> = (props) => {
       "_blank",
       "height=595,width=440,scrollbars=yes,left=0"
     ); //todo: add config
-    //if opened -> all is ok
+    
+    if (
+      windowObjectReference &&
+      isL2MoonpayNewWindowIntegration()
+    ) {
+      return replaceScreen(
+        <PaymentProgressView
+          nextStep={{
+            type: "paymentProgress",
+            progress: 80,
+            tokenIn: {
+              name: "Wrapped Ether",
+              address: "0xc778417E063141139Fce010982780140Aa0cD5Ab",
+              symbol: "WETH",
+              decimals: 18,
+              chainId: 4,
+              logoURI:
+                "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png",
```

Comment:
Temporary?

Reply:
This PR is supposed to be a draft 😅 My bad